executor/sort_test.go

```diff
@@ -119,10 +122,10 @@ func (s *testSerialSuite1) TestIssue16696(c *C) {
 		line := fmt.Sprintf("%v", row)
 		disk := fmt.Sprintf("%v", row[length-1])
 		if strings.Contains(line, "Sort") || strings.Contains(line, "HashJoin") {
-			c.Assert(strings.Contains(disk, "0 Bytes"), IsFalse)
-			c.Assert(strings.Contains(disk, "MB") ||
+			require.False(t, strings.Contains(disk, "0 Bytes"))
+			require.True(t, strings.Contains(disk, "MB") ||
```

Comment:
IIRC there is a `require.(Not)Contains` method to use.

Reply:
You are right. Fixed.